cmd/backup-manager/app/export/export.go

```diff
@@ -62,18 +62,21 @@ func (bo *Options) dumpTidbClusterData(backup *v1alpha1.Backup) (string, error)
 	if err != nil {
 		return "", err
 	}
+	if backup.Spec.Mydumper.TableRegex != nil && len(*backup.Spec.Mydumper.TableRegex) > 0 {
+		return "", fmt.Errorf("dumpling doesn't support tableRegex option, please use tableFilter instead")
+	}
```

Comment:
Should we delete `TableRegex` directly?

Reply:
No need to return an error here, I think just a warning log that this is not supported anymore is ok.